rotkehlchen/chain/ethereum/typing.py

```diff
@@ -180,9 +203,10 @@ def to_db_tuple(self) -> ValidatorDailyStatsDBTuple:
         )
 
     @classmethod
-    def deserialize_from_db(cls, entry: ValidatorDailyStatsDBTuple) -> 'ValidatorDailyStats':
+    def deserialize_from_db(cls, entry: ValidatorDailyStatsDBTupleRead) -> 'ValidatorDailyStats':
```

Comment:
not needed as I explained above. Also why did you add another completely new Tuple?